ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -753,7 +753,6 @@ tests:
     env:
       ASSISTED_CONFIG: |
         STATIC_IPS=true
-        TEST_FUNC=test_networking
```

Comment:
Why deleting this ?

Reply:
You just want to change version no ?

Reply:
By deleting this we will test STATIC_IPS=true but only with default networking instead of both networking type. I think it makes sense in my mind, if not please let me know why :)

Reply:
Oh so it will run `test_install` by default, which will use the default network type with the default openshift versions

Reply:
sorry

Reply:
That means we were creating 2 clusters? one with sdn and one with ovn?
Once this PR is merged, it would be worth to remove `PACKET_PLAN`, the medium machine would then be enough to run that test.